images_metadata.yaml

```diff
@@ -1,165 +1,219 @@
 ---
 fig_A.1:
   caption: "Entity terms"
-  source: ISO 14812, Figure A.1
+  source: ISO 14812
```

Comment:
"ISO 14812:2022" please.

Reply:
"ISO 14812:2022" is a dated reference, i.e. bound to a particular document. Without the year, "ISO 14812" is an undated reference which means "ISO 14812:yyyy" where yyyy can be any edition year. Since there is a "Figure A.1" we must use a date reference because "Figure A.1" may be different between editions.